packages/init/vite.config.ts

```diff
@@ -11,17 +17,15 @@ export default defineConfig({
 			},
 		},
 		rollupOptions: {
+			output: {
+				exports: "named",
+			},
```

Comment:
We're not moving forward with Turbo for now, do we need to backport this though @angeloashmore?

Reply:
If you still see the default/named exports warning in the build output from Rollup, then yes, it could be ported. However, it isn't necessary since it just suppresses the warning.